Source/gui/SNTBinaryMessageWindowController.m

```diff
@@ -79,88 +73,44 @@ - (void)observeValueForKeyPath:(NSString *)keyPath
   }
 }
 
-- (void)loadWindow {
-  [super loadWindow];
-  NSURL *url = [SNTBlockMessage eventDetailURLForEvent:self.event customURL:self.customURL];
-
-  if (!url) {
-    [self.openEventButton removeFromSuperview];
-  } else if (self.customURL.length == 0) {
-    // Set the button text only if a per-rule custom URL is not used. If a
-    // custom URL is used, it is assumed that the `EventDetailText` config value
-    // does not apply and the default text will be used.
-    NSString *eventDetailText = [[SNTConfigurator configurator] eventDetailText];
-    if (eventDetailText) {
-      [self.openEventButton setTitle:eventDetailText];
-      // Require the button keyEquivalent set to be CMD + Return
-      [self.openEventButton setKeyEquivalent:@"\r"];  // Return Key
-      [self.openEventButton
-        setKeyEquivalentModifierMask:NSEventModifierFlagCommand];  // Command Key
-    }
-  }
-
-  NSString *dismissButtonText = [[SNTConfigurator configurator] dismissText];
-  if (dismissButtonText.length) {
-    [self.dismissEventButton setTitle:dismissButtonText];
-  }
-
-  if (!self.event.needsBundleHash) {
-    [self.bundleHashLabel removeFromSuperview];
-    [self.hashingIndicator removeFromSuperview];
-    [self.foundFileCountLabel removeFromSuperview];
-  } else {
-    self.openEventButton.enabled = NO;
-    self.hashingIndicator.indeterminate = YES;
-    [self.hashingIndicator startAnimation:self];
-    self.bundleHashLabel.hidden = YES;
-    self.foundFileCountLabel.stringValue = @"";
-  }
+- (void)windowDidResize:(NSNotification *)notification {
+  [self.window center];
+}
 
-  if (!self.event.fileBundleName) {
-    [self.applicationNameLabel removeFromSuperview];
-  }
+- (void)showWindow:(id)sender {
+  if (self.window) [self.window orderOut:sender];
+
+  self.window =
+    [[NSWindow alloc] initWithContentRect:NSMakeRect(0, 0, 0, 0)
```

Comment:
I think we want this to be an `SNTMessageWindow` to match existing behavior (see .xib) of an NSPanel. It also gets us the fade in and out animations.

Reply:
SNTMessageWindow is gone; the fade in/out haven't been working for a while and are superfluous and the flags passed to create this window re-create a window without a title bar or traffic light buttons.

Reply:
Ah, I see. Even though the .xib refers to it, we were not compiling in `SNTMessageWindow` as of
https://github.com/google/santa/pull/1010.